pkg/config/config.go

```diff
@@ -144,29 +261,29 @@ func getEnv(key string, fallback string) string {
 }
 
 // Parse env var to int if key exists
-func getIntEnv(key string, fallback int) int {
+func getIntEnv(key string, fallback int) (int, error) {
 	envStrValue := getEnv(key, "")
 	if envStrValue == "" {
-		return fallback
+		return fallback, nil
 	}
 	envIntValue, err := strconv.Atoi(envStrValue)
 	if err != nil {
-		log.Fatalln("Env Var " + key + " must be an integer")
+		return -1, err
```

Comment:
instead of blindly passing back the err can you use `fmt.Errorf("Env var %s must be an integer: %w", key, err)` to wrap the error message to give some context where the Atoi conversion failed.

Reply:
or errors.New is fine too, like below